website/src/layout.tsx

```diff
@@ -9,13 +9,32 @@ import Footer from "./footer"
 import "./global-styles.css"
 import * as css from "./layout.css"
 import { MobileNav, NavMenu } from "./menu"
+import { HeaderPrefDrawer, selectedMode, selectedPhonetics } from "./mode"
 import { hideOnPrint, themeClass } from "./sprinkles.css"
+import { PhoneticRepresentation, ViewMode } from "./types"
 import "./wordpress.css"
 
 /* const ClientSignIn = lazy(() => import("./client/signin")) */
 
+// Set up experiencelevel
+export const preferencesContext = React.createContext({
+  expLevel: 0 as ViewMode,
+  expLevelUpdate: (p: ViewMode) => {},
+  phonRep: 0 as PhoneticRepresentation,
+  phonRepUpdate: (p: PhoneticRepresentation) => {},
```

Comment:
The common convention for state updates in React is `value` and `setValue` as opposed to what you've got here, `value` and `valueUpdate`